src/DotVVM.Framework/Resources/Scripts/DotVVM.Validation.ts

```diff
@@ -445,12 +505,14 @@ class DotvvmValidation {
     public showValidationErrorsFromServer(args: DotvvmAfterPostBackEventArgs) {
         // resolve validation target
         var context = ko.contextFor(args.sender);
-        var validationTarget: KnockoutValidatedObservable<any>
-            = dotvvm.evaluator.evaluateOnViewModel(context, args.postbackOptions.additionalPostbackData.validationTargetPath);
-        if (!validationTarget) return;
+        var validationTarget = <KnockoutObservable<any>>dotvvm.evaluator.evaluateOnViewModel(
+            context,
+            args.postbackOptions.additionalPostbackData.validationTargetPath);
```

Comment:
Could you please indent these argument more so it's clear that they are args of the `evaluateOnViewModel` method, this way it looks like a declaration of multiple variables.